bin/run-wer-automation.sh

```diff
@@ -25,7 +14,7 @@ python -u DeepSpeech.py \
   --dropout_rate 0.30 \
   --default_stddev 0.046875 \
   --checkpoint_step 1 \
-  --export_dir "${ds_dataroot}/exports/`git rev-parse --short HEAD`" \
+  --checkpoint_dir "${COMPUTE_KEEP}" \
+  --export_dir "${COMPUTE_RESULTS}" \
+  --wer_log_pattern "GLOBAL LOG: logwer('${COMPUTE_ID}', '%s', '%s', %f)"\
   "$@"
-
-ln -sf $ds_export_dir $ds_export_dir/../latest
```

Comment:
As long as we make it easy to download the models that appear on doyouunderstand.me, I'm happy :)